### ISSUE TITLE
Invalid style value in simple docx generated by MS Office 2007.

### ISSUE THREAD

**Issue by wapmorgan** [PHPOffice/PHPWord#391]
**The file I've tested that generated by original MS Office 2007 and contains only two lines of text with center and left alignment.**

I have this file (attached). I need to get sections of this file, so I use this code:

``` php
$this->wordObject = \PhpOffice\PhpWord\IOFactory::load($filename);
```

But this fails with message

> PHP Warning:  Uncaught exception 'InvalidArgumentException' with message 'Invalid style value.' ...
> Stack trace:
> Stack trace: #0  [path to vendor dir]/phpoffice/phpword/src/PhpWord/Style/NumberingLevel.php(190): PhpOffice\PhpWord\Style\AbstractStyle->setEnumVal('none', Array, NULL)

[**test.docx on dropbox**](https://www.dropbox.com/s/n7zoye5tbd0dq28/test.docx?dl=0)

**Comment by flip111** [comment]
Same problem here

**Comment by shubaivan** [comment]
I have this problem too, I have tha Cv.doc, open in Libre Office Writer and save the docx and what I have:
Invalid style value: none Options:bullet,decimal,upperRoman,lowerRoman,upperLetter,lowerLetter
at AbstractStyle ->setEnumVal ('none', array('bullet', 'decimal', 'upperRoman', 'lowerRoman', 'upperLetter', 'lowerLetter'), null) 
in vendor/phpoffice/phpword/src/PhpWord/Style/NumberingLevel.php at line 190   +

**Comment by bzhelin** [comment]
Same problem here, and can't parse word the GBK document

**Comment by benphilips** [comment]
Hmm... Seems like it's not fixed yet. I got the same problem when I tried to read a .DOCX file.
```
Invalid style value: ideographTraditional Options:bullet,decimal,upperRoman,lowerRoman,upperLetter,lowerLetter
```

When I tried to read a .DOC file, I got the following error.
```
Notice (8): Undefined property: PhpOffice\PhpWord\Shared\OLERead::$documentSummaryInformation [ROOT/Vendor/phpoffice/phpword/src/PhpWord/Reader/MsDoc.php, line 144]
```

Any fixes?

**Comment by eothein** [comment]
I have the same problem, with a very simple document

**Comment by wapmorgan** [comment]
They will never fix it. Take it easier

**Comment by eothein** [comment]
Hello Wapmorgan. How come they will not fix it? Is there a workaround? 

What do you mean with take it easier?

**Comment by wapmorgan** [comment]
This issue is 2 years old, they did nothing. So, use or replace as it is.

**Comment by troosan** [comment]
well, it's never too late, this should be fixed in 0.14
All values from http://www.datypic.com/sc/ooxml/t-w_ST_NumberFormat.html are supported

**Comment by jeffsrepoaccount** [comment]
@troosan When is 0.14 coming? This is still an issue.
